### PR TITLE
added some simple tabs

### PR DESCRIPTION
**Pull request by BakaBoing**

### CODE REVIEW

File: src/features/TaskScreen/TaskScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        <TabView
+            navigationState={{ index, routes }}
+            renderScene={renderScene}
+            onIndexChange={setIndex}
+            initialLayout={initialLayout}
+        />
+    );
+}
+
+const styles = StyleSheet.create({
```
Comment (lukasbals) [inline src/features/TaskScreen/TaskScreen.tsx]: You could create your own `styles.ts` file like in the `HomeScreen` component.

File: src/features/TaskScreen/TaskScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+const DoneRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+function TaskScreen() {
+    const [index, setIndex] = React.useState(0);
+    const [routes] = React.useState([
```
Comment (lukasbals) [inline src/features/TaskScreen/TaskScreen.tsx]: I guess you could just use a constant here without the `useState()`. 

Like that: 
```
const routes = [
  { key: 'ToDo', title: 'ToDo' },
  { key: 'InProgress', title: 'In Progress' },
  { key: 'Verify', title: 'Verify' },
  { key: 'Done', title: 'Done' },
];
```

Additionally, I would then move it up to the beginning of the file.

File: src/features/TaskScreen/TaskScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+const DoneRoute = () => (
+    // put component here, or direct feed with tasks
+    <View style={[styles.scene, { backgroundColor: '#673ab7' }]} />
+);
+
+const initialLayout = { width: Dimensions.get('window').width };
+
+function TaskScreen() {
+    const [index, setIndex] = React.useState(0);
```
Comment (lukasbals) [inline src/features/TaskScreen/TaskScreen.tsx]: If you import react like that: `import React, { useState } from 'react'` ... you would not need `React.` here.

File: src/features/TaskScreen/TaskScreen.tsx (hunk 2 above)
Comment (BakaBoing) [reply src/features/TaskScreen/TaskScreen.tsx]: Yes, they won't change.

File: src/features/TaskScreen/TaskScreen.tsx (hunk 1 above)
Comment (BakaBoing) [reply src/features/TaskScreen/TaskScreen.tsx]: Good point

**Review by lukasbals — APPROVED** [review]
Looks good 🚀

**Pull request merged**